doc/source/notebooks/advanced/variational_fourier_features.pct.py

```diff
@@ -336,7 +336,7 @@ def objective():
 opt = gpflow.optimizers.Scipy()
 opt.minimize(objective,
              variables=m.trainable_variables,
-             options=dict(maxiter=5000), method='L-BFGS-B')  # TODO: make work with BFGS
+             options=dict(maxiter=5000))
```

Comment:
Question: Is BFGS the default? So this TODO is just out of date?

Reply:
No, L-BFGS-B is the default, I think the comment was from when we thought we should get things to work with BFGS (the scipy.minimize default) but actually for functions with very parameters such as here L-BFGS-B works much better, and L-BFGS-B is now explicitly the gpflow.Scipy default. I.e., now I think it makes no sense "trying to get it to work with BFGS". L-BFGS-B is the better option for us. If it optimizes with L-BFGS-B we're done.